diff --git a/lua/modicator/init.lua b/lua/modicator/init.lua
index d1ad77e..d95cd51 100644
--- a/lua/modicator/init.lua
+++ b/lua/modicator/init.lua
@@ -142,13 +142,6 @@ function M.set_cursor_line_highlight(hl_name)
     hl = vim.tbl_extend('keep', { bg = cl.bg }, hl)
   end
   api.nvim_set_hl(0, 'CursorLineNr', hl)
-
-  local register_is_executing = vim.fn.reg_executing() ~= ""
-
-  -- Workaround for https://github.com/neovim/neovim/issues/25851
-  if not vim.o.lazyredraw and not register_is_executing then
-    vim.cmd.redraw()
-  end
 end
 
 ---@return integer augroup Augroup ID
